static/app/views/issueDetails/traceTimeline/traceTimeline.spec.tsx

```diff
@@ -259,39 +265,75 @@ describe('TraceTimeline', () => {
     expect(useRouteAnalyticsParams).toHaveBeenCalledWith({
       has_related_trace_issue: false,
       trace_timeline_status: 'empty',
-      trace_timeline_two_issues: false,
     });
   });
 
-  it('works for plans with no global-views feature', async () => {
+  it('trace timeline works for plans with no global-views feature', async () => {
+    // This test will call the endpoint without the global-views feature, thus,
+    // we will only look at the current project (project: event.projectID) instead of passing -1
     MockApiClient.addMockResponse({
       url: `/organizations/${organization.slug}/events/`,
       body: issuePlatformBody,
       match: [
         MockApiClient.matchQuery({
           dataset: 'issuePlatform',
-          // Since we don't have global-views, we only look at the current project
           project: event.projectID,
         }),
       ],
     });
     MockApiClient.addMockResponse({
       url: `/organizations/${organization.slug}/events/`,
-      body: emptyBody,
+      body: twoIssuesBody,
       match: [
         MockApiClient.matchQuery({
           dataset: 'discover',
-          // Since we don't have global-views, we only look at the current project
           project: event.projectID,
         }),
       ],
     });
 
     render(<TraceTimeline event={event} />, {
       organization: OrganizationFixture({
-        features: [], // No global-views feature
+        features: [],
       }),
     });
     expect(await screen.findByLabelText('Current Event')).toBeInTheDocument();
   });
+
+  it('trace-related issue works for plans with no global-views feature', async () => {
+    // This test will call the endpoint without the global-views feature, thus,
+    // we will only look at the current project (project: event.projectID) instead of passing -1
+    MockApiClient.addMockResponse({
+      url: `/organizations/${organization.slug}/events/`,
+      body: issuePlatformBody,
+      match: [
+        MockApiClient.matchQuery({
+          dataset: 'issuePlatform',
+          project: event.projectID,
+        }),
+      ],
+    });
+    MockApiClient.addMockResponse({
+      url: `/organizations/${organization.slug}/events/`,
+      body: discoverBody,
```

Comment:
Using `discoverBody` & `issuePlatformBody` will yield two issues, thus, trace-related issues will be rendered instead of the trace-timeline.